client/starwhale/core/eval/model.py

```diff
@@ -0,0 +1,476 @@
+        size: int = DEFAULT_PAGE_SIZE,
+    ) -> t.Tuple[t.List[t.Dict[str, t.Any]], t.Dict[str, t.Any]]:
+        _cls = cls._get_job_cls(project_uri)
+        return _cls.list(project_uri)
+
+    @classmethod
+    def get_job(cls, job_uri: URI) -> EvaluationJob:
+        _cls = cls._get_job_cls(job_uri)
+        return _cls(job_uri)
+
+
+# TODO: Storage Class Mixed
+class StandaloneEvaluationJob(EvaluationJob):
+    def __init__(self, uri: URI) -> None:
+        super().__init__(uri)
+        self.store = EvaluationStorage(uri)
+        self._sw_config = SWCliConfigMixed()
+
+    @classmethod
+    def run(
+        cls,
+        project_uri: URI,
+        model_uri: str,
+        dataset_uris: t.List[str],
+        runtime_uri: str,
+        version: str = "",
+        name: str = "",
+        desc: str = "",
+        **kw: t.Any,
+    ) -> t.Tuple[bool, str]:
+        use_docker = kw.get("use_docker", False)
+        typ = kw.get("typ", EvalTaskType.ALL)
+        step = kw.get("step", "")
+        task_index = kw.get("task_index", 0)
+
+        ee = EvalExecutor(
+            model_uri=model_uri,
+            dataset_uris=dataset_uris,
+            project_uri=project_uri,
+            runtime_uri=runtime_uri,
+            version=version,
+            name=name,
+            desc=desc,
+            gencmd=kw.get("gencmd", False),
+            use_docker=use_docker,
+        )
+        if runtime_uri and not use_docker:
+            RuntimeProcess.from_runtime_uri(
+                uri=runtime_uri,
+                target=ee.run,
+                args=(typ, step, task_index),
+                runtime_restore=kw.get("runtime_restore", False),
+            ).run()
+        else:
+            ee.run(typ, step, task_index)
+
+        return True, ee._version
+
+    def _get_report(self) -> t.Dict[str, t.Any]:
+        os.environ["SW_ROOT_PATH"] = str(self._sw_config.datastore_dir)
+        os.environ["SW_PROJECT"] = self._sw_config.current_project
+        os.environ["SW_EVAL_ID"] = self.store.id
+        logger.debug(
+            f"datastore path:{str(self._sw_config.datastore_dir)}, eval_id:{self.store.id}"
+        )
+        _datastore = wrapper.Evaluation()
+        return _datastore.get_metrics()
+        # report = {}
+        # with jsonlines.open(str(self.store.eval_report_path.resolve()), "r") as _reader:
+        #     for _report in _reader:
+        #         if not _report or not isinstance(_report, dict):
+        #             continue
+        #
+        #         report = _report
+        #         break
+        # return report
+
+    @staticmethod
+    def _do_flatten_summary(summary: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
+        rt = {}
+
+        def _f(_s: t.Dict[str, t.Any], _prefix: str = "") -> None:
+            for _k, _v in _s.items():
+                _k = f"{_prefix}{_k}"
+                if isinstance(_v, dict):
+                    _f(_v, _prefix=f"{_k}.")
+                else:
```

Comment:
need to flatten list/tuple object?

Reply:
should we use a common function to do flatten dict?

Reply:
fixed